package.json

```diff
@@ -1,99 +1,106 @@
-    "start": "sirv dist --single",
-    "validate": "svelte-check",
-    "size": "size-limit",
-    "lint": "eslint src",
-    "test": "jest src",
-    "test:watch": "npm run test -- --watch",
-    "clean": "rm -rf ./dist"
-  },
-  "engines": {
-    "node": ">= 8"
-  },
-  "browserslist": [
-    "extends browserslist-config-google"
-  ],
-  "size-limit": [
-    {
-      "limit": "180 KB",
-      "path": "dist/index.html"
-    }
-  ],
-  "devDependencies": {
-    "@babel/core": "^7.17.5",
-    "@babel/plugin-proposal-optional-chaining": "^7.11.0",
-    "@babel/plugin-transform-runtime": "^7.11.5",
-    "@babel/preset-env": "^7.11.5",
-    "@babel/runtime": "^7.11.2",
-    "@rollup/plugin-babel": "^5.2.1",
-    "@rollup/plugin-commonjs": "^15.0.0",
-    "@rollup/plugin-json": "^4.1.0",
-    "@rollup/plugin-node-resolve": "^8.0.0",
-    "@rollup/plugin-typescript": "^6.0.0",
-    "@size-limit/preset-app": "^4.6.0",
-    "@testing-library/jest-dom": "^5.11.5",
-    "@testing-library/svelte": "^3.0.0",
-    "@tsconfig/svelte": "^1.0.0",
-    "@types/jest": "^26.0.15",
-    "autoprefixer": "^10.0.1",
-    "babel-jest": "^26.6.1",
-    "browserslist-config-google": "latest",
-    "core-js": "^3.6.5",
-    "eslint": "^7.10.0",
-    "eslint-plugin-svelte3": "^3.0.0",
-    "jest": "^26.6.1",
-    "postcss": "^8.4.8",
-    "postcss-easy-import": "^4.0.0",
-    "postcss-flexbugs-fixes": "^5.0.2",
-    "postcss-load-config": "^3.0.0",
-    "postcss-preset-env": "^7.4.2",
-    "postcss-url": "^10.1.3",
-    "rollup": "^2.3.4",
-    "rollup-plugin-bundle-html-plus": "^1.4.0",
-    "rollup-plugin-copy": "^3.4.0",
-    "rollup-plugin-css-only": "^3.0.0",
-    "rollup-plugin-import-resolver": "^1.0.4",
-    "rollup-plugin-inline-svg": "^2.0.0",
-    "rollup-plugin-livereload": "^2.0.0",
-    "rollup-plugin-node-builtins": "2.0.0",
-    "rollup-plugin-node-globals": "^1.4.0",
-    "rollup-plugin-svelte": "^7.0.0",
-    "rollup-plugin-terser": "^7.0.0",
-    "rollup-plugin-visualizer": "^4.1.1",
-    "sass": "^1.44.0",
-    "size-limit": "^4.6.0",
-    "svelte": "^3.31.0",
-    "svelte-asyncable": "^2.0.4",
-    "svelte-check": "^1.0.0",
-    "svelte-code-editor": "^1.0.1",
-    "svelte-content-loader": "^1.1.3",
-    "svelte-jester": "^1.1.5",
-    "svelte-json-tree-view": "^1.0.0",
-    "svelte-media": "^0.1.5",
-    "svelte-morphing-modal": "^2.2.0",
-    "svelte-pathfinder": "^3.2.3",
-    "svelte-preprocess": "^4.0.0",
-    "svelte-spectre": "^0.2.6",
-    "ts-jest": "^26.4.1",
-    "tslib": "^2.0.0",
-    "typescript": "^3.9.3"
-  },
-  "dependencies": {
-    "copy-to-clipboard": "^3.3.1",
-    "debounce-promise": "^3.1.2",
-    "optimade": "^2.0.3",
-    "optimade-mpds-nlp": "^0.1.1",
-    "sirv-cli": "^1.0.0"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/tilde-lab/optimade.science.git"
-  }
+	"name": "optimade-esbuild",
+	"version": "1.0.0",
+	"description": "Starter for new Svelte application with ESBuild bundler ",
```

Comment:
@Valexr why the name, version and descr have been changed? The version should probably be `2.0.0`.